view/activity_activity_form.xml

```diff
@@ -45,5 +47,6 @@ contains the full copyright notices and license terms. -->
         <button name="plan"/>
         <button name="do"/>
         <button name="cancel"/>
+        <button name="createsubactivities" string="SubActivities"/>
```

Comment:
You must declare the button in activity.xml and then the string is unnecessary here.